firstvoices/backend/admin/dictionary_admin.py

```diff
@@ -77,6 +78,14 @@ class DictionaryEntryLinkInline(BaseInlineAdmin):
     fields = ("to_dictionary_entry",) + BaseInlineAdmin.fields
 
 
+class WordOfTheDayInline(BaseInlineAdmin):
+    model = WordOfTheDay
+    fields = (
+        "dictionary_entry",
+        "date",
+    ) + BaseInlineAdmin.fields
+
+
```

Comment:
Oh nice! So in a pinch, a superadmin could add a word of the day for someone on a specific date, right?

Reply:
Yes, till we have a write API for users to add a word of the day, if a community wants something specific on a date, superadmins can do that.